server/src/app.controller.ts

```diff
@@ -1,57 +1,52 @@
-import { Controller, Get, Post, Body } from '@nestjs/common';
-import { TestService, BrowseService } from './app.service';
+import { Body, Controller, Get, Post } from '@nestjs/common';
 import ApiFactory from './api/import';
-
+import * as Dtos from './app.dtos';
+import { BrowseService, TestService } from './app.service';
 
 function getApi(description: string) {
   return ApiFactory.getInstance().getApi('v1.0', description);
 }
 
-function processRequest(func: Function, body): string {
-  return JSON.stringify(func(body));
-}
 @Controller()
 export class AppController {
-  constructor(private readonly testService: TestService, 
-    private readonly browseService: BrowseService) {}
+  constructor(private readonly testService: TestService,
+    private readonly browseService: BrowseService) { }
 
   @Get(ApiFactory.getInstance().getApi('test', '/test1/test2/get'))
   getTest(): string {
     return this.testService.getTest();
   }
 
-  @Get(getApi('/browse/question-list/get'))
```

Comment:
What happened to line 23?

Reply:
Probably rebasing error, I'll add it back